### DISCUSSION TITLE
How to Set Up SparkSketch

### DISCUSSION THREAD

**Discussion by hueyee** [SparkSketch-Team/SparkSketch#22] [Announcements]
Hey gang, here are some tutorials on settings up SparkSketch

[What to Download](https://www.youtube.com/watch?v=K6i4yW7kleM)
[Git Clone](https://www.youtube.com/watch?v=eF0-LsRRzXU)
[Environment Variables](https://www.youtube.com/watch?v=1cihMxCACpw)
[API Calls](https://www.youtube.com/watch?v=u0emBCT2bjc)
[Branches](https://www.youtube.com/watch?v=ELkuGNC-_QY)

**Comment by gdiz101** [comment]
if i have node 16.20.2 and npm 8.19.4 does that work or do i need to redo?

**Reply by hueyee** [reply]
There is a tool called NVM or Node Version Manager that can manage between different versions of Node and NPM. If you don't need the older version of Node you can go to Control Panel > Programs > Uninstall > Node.js. I have never used NVM so look up a tutorial

**Reply by gdiz101** [reply]
bet ok thx. Watched all the videos super helpful I will begin asap after watching more tutorials

**Comment by hueyee** [comment]
[How to Handle Authentication Tokens](https://youtu.be/ijhGfoRrZ0o)
